pkg/ansible/run.go

```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/operator-framework/operator-sdk/pkg/metrics"
 	"github.com/operator-framework/operator-sdk/pkg/restmapper"
 	sdkVersion "github.com/operator-framework/operator-sdk/version"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
```suggestion
	corev1 "k8s.io/api/core/v1"
```

Since we have many V1, wyt about we add the pkg to make clear where it came from?

Reply:
Since this is just a backport I'd rather we not change the content of the commit, we should fix it in master if it's an issue.